packages/cli/src/commands/test.ts

```diff
@@ -108,11 +109,14 @@ export default class Test extends Command {
       }, <Record<string, Runtime>> {}),
       checklyConfigConstructs,
     })
-    const { checks: checksMap, groups: groupsMap } = project.data
-    const checks = Object.entries(checksMap)
+    const checks = Object.entries(project.checks)
       .filter(([_, check]) => {
```

Comment:
```suggestion
      .filter(([, check]) => {
```

Reply:
Good idea, we can get rid of the lint error this way.